pyanaconda/bootloader/zipl.py

```diff
@@ -162,7 +156,7 @@ def install(self, args=None):
             # exceed 896 bytes; there is nothing we can do about this, so just
             # catch the error and show it to the user instead of crashing
             elif line.startswith("Error: The length of the parameters "):
-                errorHandler.cb(ZIPLError(line))
+                raise BootLoaderError(line)
```

Comment:
Did you looked on the possible output. The problem I'm seeing here is that it could be hard to find out this is a ZIPL problem. Raising bootloader error with output of zipl command could be pretty confusing.

Reply:
The removed error handler `_ziplErrorHandler` doesn't say anything about zipl neither, so the user will see pretty much the same message as before. The example of the content of `line` is below. I'd rather not change it.

`Error: The length of parameters line (1185 bytes) exceeds the allowed maximum (896 bytes)`.

Reply:
Agree. Thanks for explanation.